app/jobs/update_allele_registry_ids.rb

```diff
@@ -2,14 +2,27 @@ class UpdateAlleleRegistryIds < ActiveJob::Base
   def perform(recurring = true)
     begin
       for v in Variant.where(allele_registry_id: nil) do
-        v.allele_registry_id = get_allele_registry_id(v)
+        allele_registry_id = get_allele_registry_id(v)
+        v.allele_registry_id = allele_registry_id
         v.save
       end
+      for v in Variant.where.not(allele_registry_id: nil) do
+        allele_registry_id = get_allele_registry_id(v)
+        add_allele_registry_link(allele_registry_id)
+      end
     ensure
       reschedule if recurring
     end
   end
 
+  def add_allele_registry_link(allele_registry_id)
+    system("ruby #{script_path} put \"reg.test.genome.network/allele/#{allele_registry_id}/externalSource/civic\" \"p1=#{allele_registry_id}\" #{allele_registry_username} #{allele_registry_password}")
```

Comment:
Is there a reason we're shelling out here instead of just calling this script directly? We could toss it in `lib` and just treat it like any other class in the app. Unless we're running it via CLI other places?